x-pack/test/case_api_integration/spaces_only/tests/trial/configure/post_configure.ts

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { ConnectorTypes } from '../../../../../../plugins/cases/common/api';
+import { FtrProviderContext } from '../../../../common/ftr_provider_context';
+import { ObjectRemover as ActionsRemover } from '../../../../../alerting_api_integration/common/lib';
+import {
+  ExternalServiceSimulator,
+  getExternalServiceSimulatorPath,
+} from '../../../../../alerting_api_integration/common/fixtures/plugins/actions_simulators/server/plugin';
+
+import {
+  getConfigurationRequest,
+  removeServerGeneratedPropertiesFromSavedObject,
+  getConfigurationOutput,
+  deleteConfiguration,
+  createConfiguration,
+  createConnector,
+  getServiceNowConnector,
+  getAuthWithSuperUser,
+} from '../../../../common/lib/utils';
+import { nullUser } from '../../../../common/lib/mock';
+
+// eslint-disable-next-line import/no-default-export
+export default ({ getService }: FtrProviderContext): void => {
+  const supertest = getService('supertest');
+  const es = getService('es');
+  const kibanaServer = getService('kibanaServer');
+  const authSpace1 = getAuthWithSuperUser();
+
+  describe('post_configure', () => {
+    const actionsRemover = new ActionsRemover(supertest);
+    let servicenowSimulatorURL: string = '<could not determine kibana url>';
+
+    before(() => {
+      servicenowSimulatorURL = kibanaServer.resolveUrl(
+        getExternalServiceSimulatorPath(ExternalServiceSimulator.SERVICENOW)
+      );
+    });
+
+    afterEach(async () => {
+      await deleteConfiguration(es);
+      await actionsRemover.removeAll();
+    });
+
+    it('should create a configuration with a mapping in space1', async () => {
```

Comment:
Maybe we should test that a user with permissions to space 1 cannot create a configuration to space2.

Reply:
Same as above, can we do that with security disabled?

Reply:
You are right! I was confused.